gradle.properties

```diff
@@ -19,7 +19,7 @@ platformVersion = 2024.1
 
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
-platformPlugins = com.intellij.java, org.jetbrains.kotlin
+platformPlugins = com.intellij.java
```

Comment:
The elephant porblem is still open

Reply:
<img width="460" alt="image" src="https://github.com/JetBrains-Research/TestSpark/assets/70476032/8e435554-bff8-4ba3-83f2-0cb93fcb9789">

In my side it was resolved

Reply:
you right, sorry